src/status_im/contexts/wallet/send/transaction_confirmation/view.cljs

```diff
@@ -125,56 +125,91 @@
           :emoji               (:emoji account)
           :customization-color (:color account)}]])]))
 
+(defn network-name-from-chain-id
+  [chain-id]
+  (let [network-name (-> (rf/sub [:wallet/network-details-by-chain-id chain-id])
+                         :network-name)]
+    (if (= network-name :mainnet) :ethereum network-name)))
```

Comment:
@OmarBasem, @smohamedjavid - don't we already have a solution for this somewhere else? 🤔

Reply:
@smohamedjavid I think Javid's PR is not merged yet